### PR TITLE
change url strings value to URL

### PR DESCRIPTION
**Pull request by Krindog7337**
Issues #134

### CODE REVIEW

**Review by nlogozzo — APPROVED** [review]

**Comment by fsobolev** [comment]
I think it should be changed from "Video URL" to "Media URL" or "Video, audio or playlist URL". As I said in some other PR and in matrix, it would be better to give users more hints that the app is capable of downloading not only youtube videos, but everything yt-dlp can get. https://github.com/nlogozzo/NickvisionTubeConverter/discussions/121#discussion-5004266 is a good evidence that "Video URL" can be confusing.

**Comment by Krindog7337** [comment]
> I think it should be changed from "Video URL" to "Media URL" or "Video, audio or playlist URL". 

> it would be better to give users more hints that the app is capable of downloading not only youtube videos, but everything yt-dlp can get.

in that case, how about giving a pop-over info on right side of text input? like on pointer assistance settings (GNOME 44)

image
image

File: NickvisionTubeConverter.Shared/Resources/Strings.de.resx [hunk 1]
```diff
@@ -334,7 +334,7 @@ marcin https://github.com/martin-desktops</value>
     <value>Wenn diese Option aktiviert ist, werden die Video-Metadaten in die heruntergeladene Datei aufgenommen.</value>
   </data>
   <data name="VideoUrl.Invalid" xml:space="preserve">
-    <value>Video Url (ungültig)</value>
+    <value>Video URL (ungültig)</value>
```
Comment (fsobolev) [inline NickvisionTubeConverter.Shared/Resources/Strings.de.resx:337]: You don't need to change anything in Strings.*.resx, Weblate will look for all changes in original Strings.resx and make all translation files match.

**Review by fsobolev — CHANGES_REQUESTED** [review]

**Comment by fsobolev** [comment]
> in that case, how about giving a pop-over info on right side of text input? like on pointer assistance settings (GNOME 44)

I like it, will implement it later 👍🏻

**Review by fsobolev — APPROVED** [review]

**Pull request merged**